CONTRIBUTING.md

```diff
@@ -407,12 +407,20 @@ RUST_LOG=debug target/debug/mirrord exec -i test -l debug -c --target pod/py-ser
 Server listening on port 80
 ```
 
-Send traffic to the Kubernetes Pod through the service
+Send traffic to the Kubernetes Pod through the service. On Linux, run the following command:
 
 ```bash
 curl $(minikube service py-serv --url)
 ```
 
+On macOS, run the command below and keep the terminal open.
```

Comment:
Oh wow, do you really have to keep the terminal open in macos to see the service url? That's so sad :disappointed:

Reply:
> ❗  Because you are using a Docker driver on darwin, the terminal needs to be open to run it
Yeah, need to keep it open or the port forwarding will stop working